accounts/abi/bind/backends/simulated_test.go

```diff
@@ -995,3 +949,84 @@ func TestSimulatedBackend_PendingAndCallContract(t *testing.T) {
 		t.Errorf("response from calling contract was expected to be 'hello world' instead received %v", string(res))
 	}
 }
+
+func TestSimulatedBackend_CallContractRevert(t *testing.T) {
```

Comment:
Would be nice to add the source contract file here. So that reader can understand the corresponding tests.